pkg/ebpf/probes/probes.go

```diff
@@ -82,51 +84,117 @@ func Init(module *bpf.Module) (Probes, error) {
 			LoadElfPhdrs:               {event: "load_elf_phdrs", probeType: kprobe, fn: "trace_load_elf_phdrs"},
 			Filldir64:                  {event: "filldir64", probeType: kprobe, fn: "trace_filldir64"},
 		},
+		netProbes: []*probe{
```

Comment:
Why not write these as standard probes?

Reply:
Yep, I thought about it.. but it felt weird to me we would have to add all the handles to each existing network event, something like:

<img width="557" alt="image" src="https://user-images.githubusercontent.com/7395852/173385728-06c11397-2d57-4374-b615-9a210329b7eb.png">

of course we can optimize it, just explaining where I went that direction, let me see what I can do.

Reply:
Anyway, I think that is the price we have to pay. Events such as DNS event wont have ICMP probes, for example. So I think we are good.